pygmt/datasets/tile_map.py

```diff
@@ -117,6 +135,15 @@ def load_tile_map(region, zoom="auto", source=None, lonlat=True, wait=0, max_ret
             "to install the package."
         )
 
+    contextily_kwargs = {}
```

Comment:
Do you want to also include `zoom`, `source`, `ll`, `wait` and `max_retries` in `contextily_kwargs`?

Reply:
Mm, not really necessary. I only made the `contextily_kwargs` dict to make things backwards compatible with `contextily<1.5.0`.